docs/reference/nodes.md

```diff
@@ -1,5 +1,65 @@
 # Shift Nodes
 
-## 1.0.0
+Nodes are the graphic units that build up the workflow. Each node is associated to an [operator](../basics/terminology/#operator), which defines the execution logic of the node, and are connected to other nodes through [plugs](../basics/terminology/#plug).
 
-To Be Defined
\ No newline at end of file
+## Node Interface
+
+There are four main interface elements that compose the node; the node's name, the operator type, the trigger plugs and the i/o's plugs.
+
+![Node UI](../../images/node_ui.png "Node UI")  
+
+* **Node Name**: Defines the name of the node. On creation, the name of the node is the name of the operator with an enumerated suffix. The name of the node can be changed through the [*Inspector Widget*](../getting_started/basics/ui_overview/#inspector).
+* **Operator Type**: The associated operator to the node. It determines the native plugs of the node.
+* **Trigger Plugs**: The trigger plugs (`>>>`) are plugs meant to define the node's position in the  execution graph. They do not pass any data. If the node has other input or output plugs connected to another node, the trigger plugs do not need to be connected; otherwise, it is recommended to connect these plugs to indicate when the node needs to get computed.

```

Comment:
```suggestion
* **Trigger Plugs**: The trigger plugs (`>>>`) are plugs meant to define the node's position in the  execution graph. They do not pass any data. If the node has other input or output plugs connected to another node, the trigger plugs do not need to be connected. Otherwise, it is recommended to connect these plugs to indicate when the node needs to get computed.

```

Reply:
Hmm why the semicolon doesn't work in this case?

Reply:
Well, in this case works it's related to the same idea, you're right, it's correct! 👍